printer_mods/README.md

```diff
@@ -66,6 +66,7 @@ like so:
 ||[Ender 3 Switchwire Y Tensioner Mod](./buzzdalf/ytension) | Y Tension Mod | :x: | :x: | :x: | :heavy_check_mark: |
  ByteChild | [ESP32 DIN Rail Bracket](./ByteChild/ESP32_DIN_Rail_Bracket) | DIN rail mounting bracket for an ESP32 | :x: | :grey_question: | :heavy_check_mark: | :x: |
  || [2-Channel Relay DIN Rail Bracket](./ByteChild/2_Channel_Relay_DIN_Rail_Bracket) | DIN Rail mounting bracket for an 2-Channel-Relay | :x: | :grey_question: | :heavy_check_mark: | :x: |
+|bythorsthunder|[MGN9_Backers_Idlers](./bythorsthunder/MGN9_Backers)|New idler uppers for MGN9 Extrusion backers |:x:|:x:|:heavy_check_mark:|:x:|
```

Comment:
Please add an `:x:` for the Trident column.

Please remove underscores inside the square brackets

    [MGN9 Backers Idlers](./bythorsthunder/MGN9_Backers)

Reply:
Changes have been completed as requested.  Thank you for taking the time to review.  As this is my first time completing a pull request I am unsure of how to proceed. Do I need to respond to each comment of yours?